app/ember/package.json

```diff
@@ -43,8 +43,9 @@
   "peerDependencies": {
     "@babel/core": "*",
     "babel-plugin-ember-modules-api-polyfill": "^2.12.0",
-    "babel-plugin-htmlbars-inline-precompile": "2 || 3",
-    "ember-source": "^3.16.0"
+    "babel-plugin-htmlbars-inline-precompile": "2 || 3 || 4",
+    "ember-cli-htmlbars": "5",
+    "ember-source": ">=3.16.0"
```

Comment:
Unsure if this change is really necessary

Reply:
I think, this can be removed? Ie. there won't be an ember project without this dependency. This MUST BE present on the project.